Source/DafnyStandardLibraries/src/dfyconfig.toml

```diff
@@ -0,0 +1,8 @@
+
+[options]
+# TODO: This isn't getting applied for some reason?
+# I'd like to use this to set a hard upper limit on brittleness
+# and avoid having to use measure-complexity or dafny-reportgenerator.
+# Leaving this set ludicrously low to show the option is broken,
+# but hoping to set this to 1000 or even 100 once it's working.
+resource-limit = 1
```

Comment:
Note that 1 still means a 1000

Reply:
```
/Users/rwillems/SourceCode/dafny/Binaries/Dafny verify --log-format=text /Users/rwillems/SourceCode/dafny/Source/DafnyStandardLibraries/src/dfyconfig.toml

Dafny program verifier finished with 4 verified, 0 errors

Results for DafnyStdLibs.Wrappers.Option.Extract (well-formedness)
  Overall outcome: Correct
  Overall time: 00:00:00.3802060
  Overall resource count: 7623
  Maximum assertion batch time: 00:00:00.0721625
  Maximum assertion batch resource count: 7623

  Assertion batch 0:
    Outcome: Valid
    Duration: 00:00:00.0721625
    Resource count: 7623

    Assertions:
      Wrappers.dfy(38,7): destructor 'value' is only applied to datatype values constructed by 'Some'

Results for DafnyStdLibs.Wrappers.Result.PropagateFailure (well-formedness)
  Overall outcome: Correct
  Overall time: 00:00:00.3731120
  Overall resource count: 8735
  Maximum assertion batch time: 00:00:00.0721694
  Maximum assertion batch resource count: 8735

  Assertion batch 0:
    Outcome: Valid
    Duration: 00:00:00.0721694
    Resource count: 8735

    Assertions:
      Wrappers.dfy(81,20): destructor 'error' is only applied to datatype values constructed by 'Failure'

Results for DafnyStdLibs.Wrappers.Result.Extract (well-formedness)
  Overall outcome: Correct
  Overall time: 00:00:00.3728960
  Overall resource count: 8627
  Maximum assertion batch time: 00:00:00.0721442
  Maximum assertion batch resource count: 8627

  Assertion batch 0:
    Outcome: Valid
    Duration: 00:00:00.0721442
    Resource count: 8627

    Assertions:
      Wrappers.dfy(88,7): destructor 'value' is only applied to datatype values constructed by 'Success'

Results for DafnyStdLibs.Wrappers.OutcomeResult.PropagateFailure (well-formedness)
  Overall outcome: Correct
  Overall time: 00:00:00.3730350
  Overall resource count: 12681
  Maximum assertion batch time: 00:00:00.0721833
  Maximum assertion batch resource count: 12681

  Assertion batch 0:
    Outcome: Valid
    Duration: 00:00:00.0721833
    Resource count: 12681

    Assertions:
      Wrappers.dfy(194,20): destructor 'error' is only applied to datatype values constructed by 'Fail''

```

Reply:
Smt log still shows:

```
...
...
(set-option :rlimit 1000)
(assert (not
 (=> (= (ControlFlow 0 0) 5) (let (($generated@@259  (=> (= (ControlFlow 0 3) (- 0 2)) ($generated@@39 $generated@@253))))
(let (($generated@@260 true))
(let (($generated@@261  (=> (and (and (= $generated@@254 ($generated@@112 $generated@@255 $generated@@256 $generated false)) ($generated@@34 $generated@@30 $generated@@253 ($generated@@31 $generated@@257) $generated@@256)) (and ($generated@@28 $generated@@257 $generated@@253) ($generated@@27 $generated@@257 $generated@@253))) (and (=> (= (ControlFlow 0 4) 1) $generated@@260) (=> (= (ControlFlow 0 4) 3) $generated@@259)))))
(let (($generated@@262  (=> (and ($generated@@35 $generated@@256) ($generated@@258 $generated@@256)) (=> (and (and ($generated@@29 $generated@@30 $generated@@253 ($generated@@31 $generated@@257)) ($generated@@34 $generated@@30 $generated@@253 ($generated@@31 $generated@@257) $generated@@256)) (and (= 1 $generated@@26) (= (ControlFlow 0 5) 4))) $generated@@261))))
$generated@@262)))))
))
(check-sat)
(pop 1)
; Valid
(get-info :rlimit)
```

Reply:
From offline conversation, it seems like this behavior for low `rlimit` values is somewhat expected. Cut an issue to follow-up on it: https://github.com/boogie-org/boogie/issues/801

For this PR its not a blocker as I've upped the limit to what I actually intended (1M), which we know from experience definitely works.